tracks/ruby/exercises/pangram/README.md

```diff
@@ -8,6 +8,22 @@ def pangram?(sentence)
 end
 ```
 
+```
+ALPHABET_26 = ("a".."z").to_a
+
+def self.pangram?(sentence)
+  text = sentence.downcase.chars
+  (ALPHABET_26 - text).empty?
+end
```

Comment:
Do we want to add a talking point about sets here? This is calculating a subset (even though it doesn't use the `Set` class). I think it's valuable to realize that this is a set problem, even if it is solved differently

Reply:
Hmm, I don't see that it needs to be in the Mentor Notes..... 
Okay if we just merge this? If you think we're missing something valuable here, we can discuss that in a separate PR?

Reply:
Sure 👍